jsFile.html

```diff
@@ -26,9 +26,26 @@
         alert("this is out of stock");
     }
 
+    //the ones below are pretty good tbh, although i have no clue why the heck one image doesnt show. 
+    //and its genuinely bothering. eitherway i went to sleep
+
+    //Had to wake up and place down let instead
+
+    // Sugar, spice, and everything nice
+    // These were the ingredients chosen
+    // To create the react framework
+    // But Professor Utonium accidentally
+    // Added an extra ingredients to the concoction--
+    // Chemical const
+    // Thus, The REACT framework was born
+    // Using their ultra-super powers
+    // Mark Zuckerberg, Jeff Bezos, and Elon Musk
+    // Have dedicated their lives to fighting crime
+    // And the forces of evil
+
     dontClickBtn.onclick = function() {
-            var myImages = new Array("/images/angular.png", "/images/file.png", "/images/smiles.png");
-            var randomNum = Math.floor(Math.random() * myImages.length);
+            let myImages = new Array("/images/angular.png", "/images/smiles.png", "/images/file.png");
```

Comment:
No need to do `new Array()` in the future. Can just use `[]`.

Reply:
Also, would `const` be better for `randomNum` as you may not want to change that variable once set?

Reply:
For the first comment, yes that makes sense I will change it to just []. Since I'm only using const once, that makes sense and I will change that too.

Reply:
Thank you! I have resolved the issues within my code.